tests/watch.test.tsx

```diff
@@ -1,12 +1,12 @@
-import { describe, expect, it, jest } from '@jest/globals'
+import { describe, expect, it, vi } from 'vitest'
+import { proxy } from 'valtio'
 import { watch } from 'valtio/utils'
-import { proxy } from 'valtio/vanilla'
```

Comment:
Does this mean we are failing to alias `valtio/vanilla`?

Reply:
Yes, it does not work with the alias trick. @arjunvegda You may be able to help.

Reply:
That's an interesting one! 😅 Based on the [type of `alias`](https://github.com/vitejs/vite/blob/21c61cbbbe1add738f3fc4e8e80de2c73b1fc5a8/packages/vite/src/types/alias.d.ts#L52-L60), it appears to be using rollup to resolve the paths, so would this do the trick? 🤔 


```diff
// vitest.config.ts
+import path from 'path'
import { defineConfig } from 'vitest/config'

export default defineConfig({
  test: {
   name: 'valtio',
    setupFiles: './tests/setup.ts',
    coverage: {
      reporter: ['text', 'json', 'html', 'text-summary'],
      reportsDirectory: './coverage/',
    },
    environment: 'jsdom',
    dir: 'tests',
    alias: [
      {
        find: /^valtio$/,
-      replacement: './src/index.ts', 
+      replacement: path.resolve(__dirname, './src/index.ts'),
      },
      {
        find: /^valtio\/(.*)$/,
-       replacement: './src/$1.ts',
+       replacement: path.resolve(__dirname, './src/$1.ts'),
      },
    ],
  },
})

```

Reply:
I guess this did the trick, thank you so much.

Reply:
hmm, but it worked for `valtio`, doesn't it? anyway, feel free to give a try.

Reply:
Yes, it's working. pushed a new change.